host/sgx/sgxquote.c

```diff
@@ -708,8 +708,9 @@ oe_result_t oe_sgx_qe_get_quote(
         }
         else // ECDSA
         {
-            // For EPID, no opt_params is taken.
-            if (opt_params || opt_params_size)
+            // For EPID, no opt_params is taken, and opt_params_size should be
+            // zero.
+            if (opt_params_size > 0)
```

Comment:
`opt_params_size` has type `size_t` thus can never be negative.

```suggestion
            if (opt_params_size)
```